x/multisig/decorator.go

```diff
@@ -22,7 +22,7 @@ func NewDecorator(auth x.Authenticator) Decorator {
 // Check enforce multisig contract before calling down the stack
 func (d Decorator) Check(ctx weave.Context, store weave.KVStore, tx weave.Tx, next weave.Checker) (weave.CheckResult, error) {
 	var res weave.CheckResult
-	newCtx, err := d.withMultisig(ctx, store, tx)
+	newCtx, err := d.authMultisig(ctx, store, tx)
```

Comment:
We could increment the cost by eg. 1 per participant in each contract referenced

Reply:
Do you mean the gas cost? If yes then I can add it.

Reply:
Yes, gas cost

Reply:
Right now there is no gas cost modification in `x/multisig`. I will add one.

Reply:
@ethanfrey implemented in https://github.com/iov-one/weave/pull/416/commits/65a530b5ca991468699edb2f85368f2dcf7246b4

Please have a look if what I did there makes sense. I had to adjust the flow a bit but all tests are passing so I guess the functionality is still correctly implemented :crossed_fingers: